README.md

```diff
@@ -1,70 +1,62 @@
-# Getting Started with Create React App
+# React-Capstone
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+> Add comments and likes to display page
+> Add tests to count for comments, likes and displaying items
+> Create HTML, CSS file
+> Create JavaScript file which manipulates the DOM; adding clicks to elements and adding fucntions to remove and item from the list
 
-## Available Scripts
+## Built With
 
-In the project directory, you can run:
+-React-js
+-Redux
 
-### `npm start`
+## Online live link
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+[Video to my demonstration](https://drive.google.com/file/d/1kzy7Sief6KqaPxV7TzpF9YeZRSjuab61/view?usp=sharing)
```

Comment:
- [ ] It is required that you deploy your application using Heroku, Netlify, or GitHub Pages and add a link to the deployed project in the README.

Reply:
please I have tried my possible best, but I can't deploy